pkg/cloudprovider/providers/azure/azure_loadbalancer.go

```diff
@@ -105,7 +127,50 @@ func (az *Cloud) EnsureLoadBalancer(clusterName string, service *v1.Service, nod
 		}
 	}
 
-	lb, lbNeedsUpdate, err := az.reconcileLoadBalancer(lb, pip, clusterName, service, nodes)
+	var lbIP *string
```

Comment:
Would it be better to move the added portion to a new func?  This one is getting so much longer with this.